2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -403,3 +403,54 @@ Il piano di progetto è il documento principale che viene redatto durante questo
+
+Tramite Miro il gruppo riesce ad avere una panoramica della situazione del lavoro, che viene poi completata da Jira, l'Issue Tracking Systema adottato. Al suo interno il gruppo ha definito le due milestone principali (RTB e PB), nelle quali ha creato delle Epic che rappresentano ciò che è necessario produrre per quella milestone, e che raccolgo tutti i task necessari alla produzione di tali prodotti (documenti o software).
+
+Milestone e Epic vengono accompagnate all'interno di Jira da una barra di completamento che rappresenta in verde il lavoro svolto, in blu il lavoro in fase di svolgimento e lascia vuota la parte dedicata alle attività definite ma non ancora in svolgimento. Queste barre contribuiscono a una rappresentazione oggettiva della situazione del progetto.
+
+Oltre a queste, Jira offre la funzionalità di visualizzazione di un burndown-chart, ovvero un grafico che rappresenta l'andamento del lavoro all'interno di uno sprint in due possibili modalità:
+
+- quantità di story points completati;
+- quantità di issue completate.
+
+Entrambe le rappresentazioni pongono nell'asse $x$ del grafico il tempo, indicato in giorni. Questi grafici contengono inoltre una retta rappresentante l'andamento ideale del lavoro, grazie alla quale risulta più semplice verificare l'efficienza del gruppo.
+
+Al termine dell'incontro di retrospettiva viene redatto il paragrafo di consuntivo dello sprint nel Piano di progetto, nel quale, oltre a fare un resoconto dello sprint, si guarda anche come sta procedendo il progetto dal punto di vista economico. Dato che ogni ruolo ha un costo, a fine sprint si calcolano i costi totali, in base alle ore che sono state svolte dai singoli ruoli in quel periodo. Questo indicatore contribuisce ad avere un resoconto completo del progetto, e permette al gruppo di comprendere meglio come sta lavorando e se sta gestendo correttamente le risorse a sua disposizione.
+
+==== Controllo
+Nell'attività di controllo si intraprendono azioni correttive nei confronti dei problemi individuati. Questi problemi possono essere di duplice natura:
+
++ mancato raggiungimento degli obiettivi prefissati;
++ miglioramenti e accortezze da adottare.
+
+===== Mancato raggiungimento degli obiettivi prefissati
+È necessario che alla chiusura dello sprint le attività ancora in fase di svolgimento vengano riportate nello sprint successivo, insieme a tutte quelle attività pianificate ma non ancora iniziate che sono considerate importanti. Tutte le attività che non vengono considerate importanti, ad esempio attività di cui si è rivalutato il grado di priorità, vengono inserite nel backlog, ovvero un archivio di task non assegnate a nessuno sprint. Una situazione di mancato raggiungimento degli obiettivi può essere causata anche da una pianificazione errata e troppo ottimista, ed è quindi necessario che essa sia rivista e migliorata.
```

Comment:
Dato che le attività da svolgere nello sprint sono provenienti di norma dal backlog, più che "inserite", direi che vengono "riportate". La specifica su cos'è il backlog è superflua in quanto sarà un termine spiegato nel glossario (rimuoverei quella riga). 
riformulerei inoltre "....obiettivi può essere causata anche da..." in "...obiettivi può essere sinonimo di...."